heclib/heclib_c/src/DssInterface/v6and7/zcopyRecord.c

```diff
@@ -125,6 +126,52 @@ int zcopyRecord(long long *ifltabFrom, long long *ifltabTo, const char *pathname
 					}
 				}
 			}
+			if (boolInternalCopy) {
+				//----------------------------------------------------------------------------//
+				// force standard copy if source and destination records are Elev time series //
+				//----------------------------------------------------------------------------//
+				char cPart[MAX_PART_SIZE];
+				zpathnameGetPart(pathnameFrom, 3, cPart, sizeof(cPart));
+				if (!strncasecmp(cPart, "ELEV", 4)) {
+					zpathnameGetPart(pathnameTo, 3, cPart, sizeof(cPart));
+					if (!strncasecmp(cPart, "ELEV", 4)) {
+						boolInternalCopy = 0; // zcopyRecordInternal doesn't copy VDI in location record
+					}
+				}
+			}
+		}
+		else if ((dataType >= DATA_TYPE_PD) && (dataType < DATA_TYPE_TEXT)) {
+			//----------------------------------------------------------------------------//
+			// force standard copy if source and destination records are Elev paired data //
+			//----------------------------------------------------------------------------//
+			int indElev = FALSE;
+			int depElev = FALSE;
+			char cPart[MAX_PART_SIZE];
+			char* saveptr;
+			char* cp;
+			zpathnameGetPart(pathnameFrom, 3, cPart, sizeof(cPart));
```

Comment:
It is not clear if the re-use of the variables indElev and depElev  for both pathnameFrom and pathnameTo processing is intentional.

If we create a function:  path_is_elevation_paired_data(const char* pathname)

would the code read this way?

```c
if( path_is_elevation_paired_data(pathnameFrom)
    && 
   path_is_elevation_paired_data(pathnameTo) )
{
boolInternalCopy = 0;
}
```

a function would make the intent clear.